Packs/AzureWAF/ReleaseNotes/1_2_11.md

```diff
@@ -0,0 +1,16 @@
+#### Modeling Rules
+
+##### Azure WAF Modeling Rule
+
+- Updated the Azure WAF Modeling Rule to support Azure WAF log types AGWAccessLogs and AGWFirewallLogs.
+
+#### Parsing Rules
+
+##### Azure WAF Parsing Rule
+
+- Updated the Azure WAF Parsing Rule to support additional timestamp formats for time field.
```

Comment:
Updated the Azure WAF parsing rule to support additional timestamp formats for the time field.